chroma-diagnostics/setup.py

```diff
@@ -12,7 +12,7 @@
     version = package_version(),
     packages = find_packages(),
     include_package_data = True,
-    author = "Intel(R) Corporation",
+    author = "Intel Corporation",
```

Comment:
Does this not need the register symbol?

Reply:
According to namesdb.intel.com:

The approved usage is: Intel Corporation

This name has been approved and adopted as a tradename.
No noun or trademark symbol (such as, ®, ™ or SM) should be used with this name.